public/app/types/settings.ts

```diff
@@ -1,6 +1,7 @@
 export type Settings = { [key: string]: SettingsSection };
 
 export type SettingsSection = Record<string, string>;
+export type VerboseSettingsSection = Record<string, Record<string, Record<string, string>>>;
```

Comment:
This is not compliant with what's defined in `auth-config/types.ts`
```suggestion
export type VerboseSettings = Record<string, Record<string, Record<string, string>>>;
```

Reply:
yea makes sense, i found a type in the types ni the auth/config for saml that we can use instead